testsuite/single_node_performance.py

```diff
@@ -784,7 +805,14 @@ def print_table(
         else:
             additional_dst_pool_accounts = 2 * MAX_BLOCK_SIZE * NUM_BLOCKS
 
-        common_command_suffix = f"{executor_type_str} {pipeline_extra_args_str} --block-size {cur_block_size} {DB_CONFIG_FLAGS} {DB_PRUNER_FLAGS} run-executor {FEATURE_FLAGS} {workload_args_str} --module-working-set-size {test.key.module_working_set_size} --main-signer-accounts {MAIN_SIGNER_ACCOUNTS} --additional-dst-pool-accounts {additional_dst_pool_accounts} --data-dir {tmpdirname}/db  --checkpoint-dir {tmpdirname}/cp"
+        data_dir = tmpdirname
+        keyless_flags = ""
+        if test.key_extra.txn_auth_mode == "keyless":
+            data_dir = keyless_tmp_dir
+            keyless_flags = "--use-keyless-accounts"
+            additional_dst_pool_accounts = 0
```

Comment:
"if keyless run, set additional_dst_pool_accounts = 0"

@igor-aptos you were suggesting this in another comment, right?